tests/spyre_util.py

```diff
@@ -226,7 +226,7 @@ def generate_cb_spyre_vllm_output(
 
         vllm_model = LLM(
             model=model,
-            tokenizer=model,
+            tokenizer="ibm-granite/granite-3.2-8b-instruct",
```

Comment:
I think this will have to be set conditionally

Reply:
Yeah just wanted to test for now